packages/babel-types/scripts/generators/docs.js

```diff
@@ -217,6 +217,8 @@ const aliasDescriptions = {
     "A cover of [Literal](https://tc39.es/ecma262/#sec-primary-expression-literals)s, [Regular Expression Literal](https://tc39.es/ecma262/#sec-primary-expression-regular-expression-literals)s and [Template Literal](https://tc39.es/ecma262/#sec-template-literals)s.",
   Loop: "A cover of loop statements.",
   Method: "A cover of object methods and class methods.",
+  Miscellaneous:
+    "A cover of non-standard AST types that are sometimes useful for development.",
```

Comment:
If we want to support `type Core = Exclude<t.Node, t.TypeScript | t.Flow>;`, why don't we add a `Core` alias? The `Miscellaneous` type is a bit vague: you don't know what is until you have checked including types.

We have `core.ts` for the spec and `experimental.ts` for proposals, since `core` types won't become experimental, we can add `Core` for `core.ts` and `CoreAndProposals` for `core.ts` and `experimental.ts`.

Reply:
Good idea. I chose `Standardized`, `Proposal`, and `StandardizedOrProposal` for the aliases.

Reply:
We should not have the `Proposal` alias, otherwise when a proposal becomes stable we'll need a breaking change to remove it from there.

Reply:
I agree with @nicolo-ribaudo. I think we can start with only `Standardized` and `TypeScript`, and add more in the future.

Reply:
> We cannot have the Proposal alias, otherwise when a proposal becomes stable we'll need a breaking change to remove it from there.

That makes sense, I'll remove.

> I think we can start with only Standardized and TypeScript, and add more in the future.

I'm gonna keep `Flow`, but agreed.